### ISSUE TITLE
Windows crosstool failure

### ISSUE THREAD

**Issue by kchodorow** [bazelbuild/bazel#1732]
See http://ci.bazel.io/view/Bazel%20bootstrap%20and%20maintenance/job/Bazel/JAVA_VERSION=1.7,PLATFORM_NAME=windows-x86_64/792/console:

```
[31m[1mERROR: [0min target '//external:cc_toolchain': no such package '@local_config_cc//': Traceback (most recent call last):
	File "C:/bazel_ci/temp/bazel_11Bpb0eX/out/external/bazel_tools/tools/cpp/cc_configure.bzl", line 447
		_find_vs_path(repository_ctx)
	File "C:/bazel_ci/temp/bazel_11Bpb0eX/out/external/bazel_tools/tools/cpp/cc_configure.bzl", line 385, in _find_vs_path
		_execute(repository_ctx, [bash_bin, "-c", "...])
	File "C:/bazel_ci/temp/bazel_11Bpb0eX/out/external/bazel_tools/tools/cpp/cc_configure.bzl", line 93, in _execute
		auto_configure_fail(result.stderr)
	File "C:/bazel_ci/temp/bazel_11Bpb0eX/out/external/bazel_tools/tools/cpp/cc_configure.bzl", line 50, in auto_configure_fail
		fail("
%sAuto-Configuration Error:%s ...))

[0;31mAuto-Configuration Error:[0m java.io.IOException: Environment variable names must not contain '='
```

I'm guessing this is 180d1b56b8aea8fab974cfa57e9182598990710c, but haven't verified, feel free to reassign.

**Comment by meteorcloudy** [comment]
I wrote the auto configuration for MSVC, I will look into it.

**Comment by laszlocsomor** [comment]
Maybe related: https://github.com/bazelbuild/bazel/issues/1736 ?

**Comment by meteorcloudy** [comment]
I already sent a CL to fix this issue, but I also came across #1736 during bootstrap after that.
